packages/core/src/dialects/abstract/query-generator.d.ts

```diff
@@ -214,4 +214,6 @@ export class AbstractQueryGenerator extends AbstractQueryGeneratorTypeScript {
    * @param bind A mutable object to which bind parameters will be added.
    */
   bindParam(bind: Record<string, unknown>): (newBind: unknown) => string;
+  showTablesQuery(schemaName?: string): string;
+  showTablesQuery(database: string, options: EscapeOptions): string;
```

Comment:
this second signature is not necessary, `database` is a misnomer as it's actually the schema too. Snowflake accepts an options bag but it's not necessary and can be removed

Reply:
Removed this.